go.sum

```diff
@@ -15,6 +15,14 @@ contrib.go.opencensus.io/exporter/prometheus v0.1.0/go.mod h1:cGFniUXGZlKRjzOyuZ
 contrib.go.opencensus.io/exporter/stackdriver v0.6.0/go.mod h1:QeFzMJDAw8TXt5+aRaSuE8l5BwaMIOIlaVkBOPRuMuw=
 contrib.go.opencensus.io/exporter/zipkin v0.1.1/go.mod h1:GMvdSl3eJ2gapOaLKzTKE3qDgUkJ86k9k3yY2eqwkzc=
 fortio.org/fortio v1.3.0/go.mod h1:Go0fRqoPJ1xy5JOWcS23jyF58byVZxFyEePYsGmCR0k=
+git.blindage.org/21h/hcloud-dns v0.0.0-20200506180043-6c1878aeb13b h1:MNEkBjEIXru4KEar5lEQUzS5IWPe2kiocaJHO8aH08E=
+git.blindage.org/21h/hcloud-dns v0.0.0-20200506180043-6c1878aeb13b/go.mod h1:xjcUpOv7TY3agUKC3jdPHR3e77ojEwdNMu4dvPmsCYo=
+git.blindage.org/21h/hcloud-dns v0.0.0-20200507105038-1c05e3e73856 h1:/4KRxksFpyaOnEKlZgcRF4fEaQdURLHDm7AtJ2fyndM=
+git.blindage.org/21h/hcloud-dns v0.0.0-20200507105038-1c05e3e73856/go.mod h1:xjcUpOv7TY3agUKC3jdPHR3e77ojEwdNMu4dvPmsCYo=
+git.blindage.org/21h/hcloud-dns v0.0.0-20200507111022-02050bb3a364 h1:XL77s6YAONMeDmByiWXeeAEJUogPW/vJcmt8FvLntUU=
+git.blindage.org/21h/hcloud-dns v0.0.0-20200507111022-02050bb3a364/go.mod h1:xjcUpOv7TY3agUKC3jdPHR3e77ojEwdNMu4dvPmsCYo=
+git.blindage.org/21h/hcloud-dns v0.0.0-20200507195107-b8e668eab385 h1:OgpuaGZsfYkxZG/L1hRFHCfGcIsCrAY7/7uCciOvkJU=
+git.blindage.org/21h/hcloud-dns v0.0.0-20200507195107-b8e668eab385/go.mod h1:xjcUpOv7TY3agUKC3jdPHR3e77ojEwdNMu4dvPmsCYo=
```

Comment:
@21h There multiple versions of same library in the go.sum. It happens because you do development of the library and test changes in the project in concurent. To remove duplication there is a command: `go mod tidy`.